server/api/index.js

```diff
@@ -0,0 +1,64 @@
+const express = require("express");
+const mongoose = require("mongoose");
+const Recipe = require("./models/recipe");
+
+const { ApolloServer, gql } = require("apollo-server-express");
+
+const typeDefs = gql`
+  input RecipeInput {
+    servings: Int,
+    prepTime: String,
+    cookTime: String,
+    ingredients: [IngredientInput],
+    steps: [String],
+    links: [String],
+    name: String
+  }
+  input IngredientInput {
+    measurement: String!,
+    quantity: Int!,
+    name: String!
+  }
+  type Ingredient {
+    measurement: String!,
+    quantity: Int!,
+    name: String!
+  }
+  type Recipe {
+    servings: Int!,
+    prepTime: String,
+    cookTime: String,
+    ingredients: [Ingredient!],
+    steps: [String!],
+    links: [String],
+    name: String!
+  }
+  type Query { recipes: [Recipe] }
+  type Mutation { createRecipe(input: RecipeInput): Recipe }
+`;
+
+const resolvers = {
+    Query: { 
+      recipes: async() => {
+        return await Recipe.find({});
+      } 
+    },
+    Mutation: {
+      createRecipe: async(parent, args) => {
+        return await Recipe.create(args.input);
+      }
+    }
+};
+
+mongoose.connect("mongodb://server:42pickleheads@ds363118.mlab.com:63118/pickleheads-cookbook")
+
+const app = express();
+const server = new ApolloServer({
+  typeDefs,
+  resolvers,
+  introspection: true,
+  playground: true,
```

Comment:
Not sure that we need these anymore

Reply:
I might leave these commented out so that we can switch the playground functionality on/off for debugging, what do you think?

Reply:
I'm fine to keep it for now. Just figured it's worth making note of it